geom2graph failed to build on Ubuntu 22.04

```
In file included from /home/nots/Documents/generative/tools/geom2graph/depends/googletest/googletest/include/gtest/internal/gtest-internal.h:65,
                 from /home/nots/Documents/generative/tools/geom2graph/depends/googletest/googletest/include/gtest/gtest.h:62,
                 from /home/nots/Documents/generative/tools/geom2graph/tests/geometry-noder-tests.cpp:10:
/home/nots/Documents/generative/tools/geom2graph/depends/googletest/googletest/include/gtest/gtest-message.h: In instantiation of ‘testing::Message& testing::Message::operator<<(const T&) [with T = std::unique_ptr<geos::geom::Geometry>]’:
/home/nots/Documents/generative/tools/geom2graph/tests/geometry-noder-tests.cpp:156:65:   required from here
/home/nots/Documents/generative/tools/geom2graph/depends/googletest/googletest/include/gtest/gtest-message.h:129:10: error: no match for ‘operator<<’ (operand types are ‘std::__cxx11::basic_stringstream<char>’ and ‘const std::unique_ptr<geos::geom::Geometry>’)
  129 |     *ss_ << val;
      |     ~~~~~^~~~~~


...


/usr/include/c++/11/ostream:750:5: error: no type named ‘type’ in ‘struct std::enable_if<false, void>’
In file included from /home/nots/Documents/generative/tools/geom2graph/depends/googletest/googletest/include/gtest/gtest.h:55,
                 from /home/nots/Documents/generative/tools/geom2graph/tests/geometry-noder-tests.cpp:10:
```

--- a/build.rs
+++ b/build.rs
@@ -8,9 +8,8 @@ fn main() {
         std::env::var("GENERATIVE_CARGO_ENABLE_CMAKE_PCH").unwrap_or_else(|_| "OFF".to_string());
     let enable_lto =
         std::env::var("GENERATIVE_CARGO_ENABLE_CMAKE_LTO").unwrap_or_else(|_| "OFF".to_string());
-    // TODO: https://github.com/Notgnoshi/generative/issues/124
     let enable_tests =
-        std::env::var("GENERATIVE_CARGO_ENABLE_CMAKE_TESTS").unwrap_or_else(|_| "OFF".to_string());
+        std::env::var("GENERATIVE_CARGO_ENABLE_CMAKE_TESTS").unwrap_or_else(|_| "ON".to_string());
     if enable_cmake_build.is_empty() {
         return;
     }

--- a/tests/geometry-noder-tests.cpp
+++ b/tests/geometry-noder-tests.cpp
@@ -10,7 +10,6 @@
 #include <gtest/gtest.h>
 
 using generative::io::from_wkt;
-using generative::io::operator<<;
 
 TEST(GeometryNoderTests, DisjointPoint)
 {
@@ -153,7 +152,7 @@ TEST(GeometryNoderTests, SnappingNoder)
     // std::cerr << noded->toString() << std::endl;
 
     EXPECT_EQ(noded->getGeometryType(), expected->getGeometryType());
-    EXPECT_TRUE(noded->equals(expected.get())) << "actual: " << noded << " expected: " << expected;
+    EXPECT_TRUE(noded->equals(expected.get()));
 }
 
 //! @todo Use google benchmark.

